src/Kanban/index.js

```diff
@@ -72,9 +67,15 @@ class Kanban extends PureComponent {
     super(props);
 
     this.state = {
-      lists: props.lists
+      lists: props.lists,
     };
 
+    this.kanbanCache = new CellMeasurerCache({
```

Comment:
Did we also need to clear this on componentDidUpdate like in the SortableList?

Reply:
Ah...yes, it probably should get cleared. I think it wasn't making a difference before because everything was fixed and the heights were always the rowHeight provided by the Kanban's parent AutoSizer.

But after making that comment above I'm thinking maybe it's better to remove this kanbanCache